iib/workers/tasks/build_regenerate_bundle.py

```diff
@@ -457,7 +484,7 @@ def _replace_csv_pullspecs(bundle_metadata, replacement_pullspecs):
     for old_pullspec, new_pullspec in replacement_pullspecs.items():
         log.debug(
             '%s will be replaced with %s in the bundle CSVs',
-            old_pullspec.to_str(),
+            old_pullspec.__repr__(),
```

Comment:
Could you please change this back to as it was. If you want this change to come in, could you please add it in a different PR

Reply:
I believe this is still pending? We discussed to remove __repr__ here

Reply:
Should be okay now.